solr/contrib/ltr/README.md

```diff
@@ -198,19 +198,19 @@ The majority of features should be possible to create using the methods describe
 above.
 
 # Defining Models
-Currently the Learning to Rank plugin supports 2 main types of
-ranking models: [Ranking SVM](http://www.cs.cornell.edu/people/tj/publications/joachims_02c.pdf)
-and [LambdaMART](http://research.microsoft.com/pubs/132652/MSR-TR-2010-82.pdf)
+Currently the Learning to Rank plugin supports 2 generalized forms of
+models: 1. Linear Model i.e. [Linear](http://www.cs.cornell.edu/people/tj/publications/joachims_02c.pdf), [Pranking](https://papers.nips.cc/paper/2023-pranking-with-ranking.pdf)
+and 2. Multiple Additive Trees i.e. [Multiple Additive Trees](http://research.microsoft.com/pubs/132652/MSR-TR-2010-82.pdf), [Gradient Boosted Regression Trees (GBRT)](https://papers.nips.cc/paper/3305-a-general-boosting-method-and-its-application-to-learning-ranking-functions-for-web-search.pdf)
 
-### Ranking SVM
-Currently only a linear ranking svm is supported. Use LambdaMART for
+### Linear
+Use Linear for a linear model. Use Multiple Additive Trees for
```

Comment:
You can delete these first 2 sentences since that's kinda covered in the section above.  I would like to see RankSVM mentioned somewhere here or above as a concrete model you can train and use here.  Should we change the link text [Linear] above to be Ranking SVM?
I prefer for us to mention concrete examples of things people can use earlier rather than later in the comments/documentation